<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Account</class>
 <widget class="QWidget" name="Account">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>找回密码</string>
  </property>
  <property name="windowIcon">
   <iconset resource="lib.qrc">
    <normaloff>:/lib/System.png</normaloff>:/lib/System.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="AcMovieLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>222</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>150</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel
{
	font-size:30px;
	color:white;
}</string>
           </property>
           <property name="text">
            <string>注册账号
提示:账号由纯数字组成
密码由大小写字母及数字组成</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>80</number>
            </property>
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">QLabel
{
	font-size:20px;
	color:white;
}</string>
              </property>
              <property name="text">
               <string>账号:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">QLabel
{
	font-size:20px;
	color:white;
}</string>
              </property>
              <property name="text">
               <string>密码:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="lineEdit">
              <property name="styleSheet">
               <string notr="true">QLineEdit
{
	color:white;
	background-color:transparent;
	border-style:solid;
	border-color:rgb(200,200,200);
	border-top-width:0px;
	border-left-width:0px;
	border-right-width:0px;
	border-bottom-width:1px;
}
QLineEdit:hover
{
	color:white;
	background-color:transparent;
	border-style:solid;
	border-color:rgb(170,170,170);
	border-top-width:0px;
	border-left-width:0px;
	border-right-width:0px;
	border-bottom-width:1px;
}
QLineEdit:focus
{
	color:white;
	background-color:transparent;
	border-style:solid;
	border-color:rgb(0, 170, 255);
	border-top-width:0px;
	border-left-width:0px;
	border-right-width:0px;
	border-bottom-width:1px;
}
</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="lineEdit_2">
              <property name="styleSheet">
               <string notr="true">QLineEdit
{
	color:white;
	background-color:transparent;
	border-style:solid;
	border-color:rgb(200,200,200);
	border-top-width:0px;
	border-left-width:0px;
	border-right-width:0px;
	border-bottom-width:1px;
}
QLineEdit:hover
{
	color:white;
	background-color:transparent;
	border-style:solid;
	border-color:rgb(170,170,170);
	border-top-width:0px;
	border-left-width:0px;
	border-right-width:0px;
	border-bottom-width:1px;
}
QLineEdit:focus
{
	color:white;
	background-color:transparent;
	border-style:solid;
	border-color:rgb(0, 170, 255);
	border-top-width:0px;
	border-left-width:0px;
	border-right-width:0px;
	border-bottom-width:1px;
}
</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLineEdit" name="AgainLine">
              <property name="styleSheet">
               <string notr="true">QLineEdit
{
	color:white;
	background-color:transparent;
	border-style:solid;
	border-color:rgb(200,200,200);
	border-top-width:0px;
	border-left-width:0px;
	border-right-width:0px;
	border-bottom-width:1px;
}
QLineEdit:hover
{
	color:white;
	background-color:transparent;
	border-style:solid;
	border-color:rgb(170,170,170);
	border-top-width:0px;
	border-left-width:0px;
	border-right-width:0px;
	border-bottom-width:1px;
}
QLineEdit:focus
{
	color:white;
	background-color:transparent;
	border-style:solid;
	border-color:rgb(0, 170, 255);
	border-top-width:0px;
	border-left-width:0px;
	border-right-width:0px;
	border-bottom-width:1px;
}
</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="Again">
              <property name="styleSheet">
               <string notr="true">QLabel
{
	font-size:20px;
	color:white;
}</string>
              </property>
              <property name="text">
               <string>重复密码:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	font-size:15px;
	color:white;
	border-style:none;
	border-radius:8px 8px;
	background-color:rgb(0, 170, 255);
}
QPushButton:hover
{
	font-size:15px;
	color:white;
	border-style:none;
	border-radius:8px 8px;
	background-color:rgb(0, 213, 255);
}
QPushButton:pressed
{
	font-size:15px;
	color:white;
	border-style:none;
	border-radius:8px 8px;
	background-color:rgb(0, 85, 255);
}</string>
           </property>
           <property name="text">
            <string>注册</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>222</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>30</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>652</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	font-size:15px;
	color:white;
	border-style:none;
	border-radius:8px 8px;
	background-color:rgb(0, 170, 255);
}
QPushButton:hover
{
	font-size:15px;
	color:white;
	border-style:none;
	border-radius:8px 8px;
	background-color:rgb(0, 213, 255);
}
QPushButton:pressed
{
	font-size:15px;
	color:white;
	border-style:none;
	border-radius:8px 8px;
	background-color:rgb(0, 85, 255);
}</string>
         </property>
         <property name="text">
          <string>返回</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="lib.qrc"/>
 </resources>
 <connections/>
</ui>
